launcher/src/backend/ethereum-services/NimbusBeaconService.js

```diff
@@ -56,6 +56,7 @@ export class NimbusBeaconService extends NodeService {
         '--keymanager-address=0.0.0.0',
         '--keymanager-token-file=/opt/app/validators/api-token.txt',
         '--jwt-secret=/engine.jwt',
+        '--history=Prune',
```

Comment:
`It is possible to switch between prune and archive modes.`
pls keep the settings lower-case